build/charts/theia/README.md

```diff
@@ -18,6 +18,14 @@ Kubernetes: `>= 1.16.0-0`
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
+| clickhouse.cluster.enable | bool | `false` | Determine whether to deploy ClickHouse as a cluster. |
```

Comment:
While we discuss on whether we consider non-clustered supported, do you think we can enable clustering by default?

Reply:
+1

Reply:
As we plan to deprecate the non-clustered, I think it might be good to remove `cluster.enable` here and always deploy ClickHouse cluster. Updated as this way.